docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -367,12 +367,24 @@ statements, the connector supports the following features:
 - {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table`, see also {ref}`delta-lake-alter-table`
```

Comment:
This list should be cleaned out and content moved to the delta-lake-data-management section .. or that section overview should be absorbed into here

Reply:
I think I should do this in a different PR.

Reply:
ok